app/models/collection_template.rb

```diff
@@ -278,6 +278,13 @@ def graph_name
     'graph'
   end
 
+  def graph_url
+    file_path = File.join('spec/fixtures/data',
```

Comment:
Probably misleading for the future to call it `graph_url` instead of something like `graph_file_path` or `graph_file`.